### DISCUSSION TITLE
Show absolute number as well as proportion

### DISCUSSION THREAD

**Discussion by Jmuccigr** [hodcroftlab/covariants#293] [Ideas]
Would it be possible to allow the variant country plots to flip between the current proportional view of variants and a view that shows the absolute numbers? This would allow, among other things, the viewing of a rise in cases as a rise in a new variant (or not).

**Comment by emmahodcroft** [comment]
Hi @Jmuccigr - Thanks for the message! We've been thinking in exactly the same direction. We won't be making this a feature on the Per Country page, but we're hoping to add a new page of 'Cases' which will show the breakdown of cases by variants - hopefully pretty soon!

**Reply by Jmuccigr** [reply]
Thanks. That sounds like it would do it, if those plots can be by-country. (I'm right now interested in the recent rises in Europe, which seem to be attributable to BA.2.)
